src/main/frontend/handler/events.cljs

```diff
@@ -298,6 +299,18 @@
         (when (and pending? (seq (state/all-available-coming-updates)))
           (plugin/open-waiting-updates-modal!))))))
 
+(defmethod handle :backup/broken-config [[_ repo content]]
+  (when (and repo content)
+    (let [path (config/get-config-path)
+          broken-path (str path "-broken")]
```

Comment:
`broken-config.edn` could be nice if you want users to have a meaningful file extension